### ISSUE TITLE
model for street view 

### ISSUE THREAD

**Issue by ZIKO94ZIKO** [chandrikadeb7/Face-Mask-Detection#4]
Hi @chandrikadeb7  , i need dataset or model support street view detection mask or drone view . any help you can provide would be greatly appreciated. thanks

**Comment by chandrikadeb7** [comment]
> Hi @chandrikadeb7 , i need dataset or model support street view detection mask or drone view . any help you can provide would be greatly appreciated. thanks

You can use Bing Search API to collect relevant images.

**Comment by ZIKO94ZIKO** [comment]
yes , agree with you , but im asking for provide a dataset already prepared , please if u have one please send it , thank you

**Comment by chandrikadeb7** [comment]
> yes , agree with you , but im asking for provide a dataset already prepared , please if u have one please send it , thank you

https://github.com/chandrikadeb7/Face-Mask-Detection/tree/master/dataset
